net/queries/auth/useQrAuth.ts

```
@@ -0,0 +1,28 @@
+		try {
+			return tokenPairFromResponse(res)
+		}catch(error){
+			throw error
+		}
+	})
+	.postProcess(({ accessToken }) => {
+    if(!accessToken) throw new Error("access token not found")
+		try{
+			saveIdFromToken(accessToken);
+		}catch(error){
+			throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)